domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -576,22 +576,6 @@ private void logPostgresConfig() {
     log.info("postgreSQLClientConfig = " + passwordRedacted.encode());
   }
 
-  /**
-   * Get connection configuration.
-   * The following properties are returned (some of which are optional):
-   * username, password, host, port, database, connectionReleaseDelay, maxPoolSize.
-   * Originally based on driver
-   * <a href="https://vertx.io/docs/vertx-mysql-postgresql-client/java/#_configuration">
-   *   Configuration
-   *   </a>.
-   * which is no longer in actual use.
-   *
-   * @return
-   */
-  public JsonObject getConnectionConfig(){
-    return postgreSQLClientConfig;
-  }
-
```

Comment:
Can you merge latest master into your branch?

Reply:
I am sorry, I have some problems with that. I created another PR https://github.com/folio-org/raml-module-builder/pull/690

Reply:
I will close this one.